### PR TITLE
chore(ONYX-1108): retrieve swa specialists data from MP

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [ONYX-1108] 

Currently, we fetch the specialists data from S3. Let’s query this data from MP instead. 

Related MP PR: [feat(onyx-878): Schema for Specialist bios by nickskalkin · Pull Request #5653 · artsy/metaphysics](https://github.com/artsy/metaphysics/pull/5653)

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- use MP swa specialists data -daria

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-1108]: https://artsyproduct.atlassian.net/browse/ONYX-1108?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a58cb9eb079957b4071cae2ef35fae4ab13344a0
</p>

File: src/app/Scenes/SellWithArtsy/Components/MeetTheSpecialists.tsx [hunk 1]
```diff
@@ -78,95 +150,6 @@ export const MeetTheSpecialists: React.FC<{
   )
 }
 
-interface SpecialistProps {
-  specialist: SpecialistsData
-  onInquiryPress: InqueryPress
-}
-
-const Specialist: React.FC<SpecialistProps> = ({ specialist, onInquiryPress }) => {
```
Comment (MounirDhahri) [inline src/app/Scenes/SellWithArtsy/Components/MeetTheSpecialists.tsx:86]: **thought:** Is there a particular need for this component to be moved within the `Flatlist`? the separation we had was nice

**Review by MounirDhahri — APPROVED** [review]
LGTM - just one comment

File: src/app/Scenes/SellWithArtsy/Components/MeetTheSpecialists.tsx (hunk 1 above)
Comment (dariakoko) [reply src/app/Scenes/SellWithArtsy/Components/MeetTheSpecialists.tsx:86]: Brought `Specialist` component back 👍

**Comment by dblandin** [comment]
Awesome 🚀

**Pull request merged**